float32 forward_white_1
float32 forward_white_2
float32 forward_yellow_1
float32 forward_yellow_2
float32 forward_stats
